release/mk-release-snapshots.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Publish the latest development release (a.k.a. daily snapshot)
+#
+
+. ~/.server.config || exit $?
+
+# auto-detect the latest 'major' version number
+SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
+
+# Hostname of CI build server
```

Comment:
I recommend making this description more precise because many CI servers build something:

```suggestion
# Hostname of the CI server where snapshots are built
```